tests/x11regressions/libreoffice/libreoffice_open_specified_file.pm

```diff
@@ -35,8 +36,10 @@ sub run() {
         type_string "/home/$username/Documents/ooo-test-doc-types/test.$tag\n";
         wait_still_screen 3;
         assert_screen("libreoffice-test-$tag", 90);
+        # Close every 3 files to reduce the VM's burden
+        if ($i % 3 == 0) { send_key_until_needlematch('libreoffice-test-doc', 'alt-f4', 3, 5); }
```

Comment:
Why not just close all files ? It will make code cleaner and also it is good practice when test clean up after itself.